bin/test_all_versions

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
this is just for testing locally.  it won't try all the ruby versions, but at least it'll go through each ActiveRecord version.  it's much more likely that we have a bug specific to an AR version than a ruby version